Frontend/src/admin/stations-tab/StationsTab.tsx

```diff
@@ -8,19 +8,23 @@ import useRefresh from './../../common/hooks/useRefresh';
 import { useSnackbar } from './../../common/hooks/useSnackbar';
 import SnackBar from '../../common/components/SnackBar';
 import { Button } from '@material-ui/core';
+import { StationStatus } from '../../common/api/models/station';
 
-enum StationFilterType {
+enum StationStatusExtension {
   All = 'all',
-  Blocked = 'blocked',
-  Active = 'active',
 }
 
+type StationFilterType = StationStatus | StationStatusExtension;
+const stationFilterType = { ...StationStatus, ...StationStatusExtension };
```

Comment:
Nie tak się robi typy, jak na moje oko

Reply:
Chyba, że to nie typ miał być, ale coś innego, ale w takim razie nie czaję co

Reply:
Hmm, chyba czaję jednak, ale to jest jakieś dziwne, a przynajmniej słabo nazwane

Reply:
Tu w gruncie rzeczy jest nietypowa sytuacja, że chcę rozszerzyć enuma o jedną wartość. Jak masz elegantsze rozwiązanie to przygarnę

Reply:
Nazwę zostawiłem taką, jaka była. De facto pasuje, bo to jest rodzaj (czyli typ) filtrowania. Zamiast `*Type` można ewentualnie `*Kind`, ale chyba nie będzie lepiej w taki sposób

Reply:
Może masz rację, *Type zabrzmiało mi jakoś tak węgiersko